integration_test/manifests/backend/ing.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: networking.k8s.io/v1beta1
+kind: Ingress
+metadata:
+  annotations:
+    nginx.ingress.kubernetes.io/rewrite-target: /$2
+  name: reporting-hub-bop-role-ui
+spec:
+  rules:
+  - http:
+      paths:
+      - path: /()(.*)
+        backend:
+          serviceName: reporting-hub-bop-role-ui
```

Comment:
What is the reason in defining the K8S manifests for ingress manually instead of using the one from helm chart?

Reply:
I followed the test harness structure as much as possible, this is just how the ingress was laid out. 
I will revisit the harness in the future.